spinny_gun.py

```diff
@@ -33,6 +33,14 @@
 MEDIUM_TEXT = pygame.font.Font("freesansbold.ttf", 30)
 SMALL_TEXT = pygame.font.Font("freesansbold.ttf", 20)
 
+# Sound Effects
+shootFx = pygame.mixer.Sound(
```

Comment:
```suggestion
SHOOT_FX = pygame.mixer.Sound(
```

Can we change all these globals to capitalized snake case?